src/app/Instance/CreateInstance/CreateInstance.tsx

```diff
@@ -104,6 +138,16 @@ const CreateInstance = (props: CreateInstanceProps): JSX.Element => {
       ]}
     >
       <Form id={FORM_ID} onSubmit={onSubmit}>
+        {(error || genericError) && (
```

Comment:
Not sure what's the cause or if you are still working on it, but there's a bug on the existing name error.

If you open the dialog, type an existing name and submit, the error doesn't show up. It does if you click the submit button again.

Reply:
Thanks for catching it. I'm going to find the root cause for this regression